concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -0,0 +1,901 @@
+      - smoke-tests
+      - acceptance-tests
+      - custom-acceptance-tests
+  - name: bosh
+    jobs:
+      - init
+      - bosh-terraform
+      - generate-bosh-config
+      - bosh-deploy
+  - name: cloudfoundry
+    jobs:
+      - generate-cf-secrets
+      - cf-terraform
+      - generate-cf-config
+      - cf-deploy
+      - smoke-tests
+      - acceptance-tests
+      - custom-acceptance-tests
+
+resources:
+  - name: pipeline-trigger
+    type: semver-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      key: {{pipeline_trigger_file}}
+
+  - name: paas-cf
+    type: git
+    source:
+      uri: https://github.com/alphagov/paas-cf.git
+      branch: {{branch_name}}
+
+  - name: vpc-tfstate
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: vpc.tfstate
+
+  - name: concourse-tfstate
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      versioned_file: concourse.tfstate
+      region_name: eu-west-1
+
+  - name: bosh-tfstate
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: bosh.tfstate
+
+  - name: bosh-secrets
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: bosh-secrets.yml
+
+  - name: bosh-CA
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: bosh-CA.tar.gz
+
+  - name: bosh-manifest
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: bosh-manifest.yml
+
+  - name: bosh-init-state
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: bosh-manifest-state.json
+
+  - name: ssh-private-key
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      versioned_file: id_rsa
+      region_name: {{aws_region}}
+
+  - name: cf-tfstate
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      versioned_file: cf.tfstate
+      region_name: eu-west-1
+
+  - name: cf-secrets
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: cf-secrets.yml
+
+  - name: cf-certs
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: cf-certs.tar.gz
+
+  - name: cf-manifest
+    type: s3-iam
+    source:
+      bucket: {{state_bucket}}
+      region_name: {{aws_region}}
+      versioned_file: cf-manifest.yml
+
+  - name: cf-release
+    type: git
+    source:
+      uri: https://github.com/cloudfoundry/cf-release
+      tag_filter: {{cf-release-version}}
+
+jobs:
+  - name: init
+    serial_groups: [bosh-deploy]
+    serial: true
+    plan:
+      - get: paas-cf
+        trigger: {{auto_deploy}}
+      - task: bootstrap-s3-state
```

Comment:
Do you think it's worth to split this into several tasks based on what we init and aggregate them. This should look better in UI and speed up initialisation.